data/json/mapgen/geany_run_script.bat

```diff
@@ -0,0 +1,6 @@
+./mortuary

```

Comment:
I tried commenting on this before, but I don't see the comment... I don't know how that got in there, no, I don't want to add that. I'll look into fixing that.